src/apps/engine/src/main.cpp

```diff
@@ -119,8 +119,12 @@ void HandleWindowEvent(const storm::OSWindow::Event &event)
 int main(int argc, char *argv[])
 {
     // Prevent multiple instances
+#ifdef _WIN32 // CreateEventA
     if (!CreateEventA(nullptr, false, false, "Global\\FBBD2286-A9F1-4303-B60C-743C3D7AA7BE") ||
         GetLastError() == ERROR_ALREADY_EXISTS)
+#else
+    if (false)
```

Comment:
the whole if-block can be surrounded with `ifdef/endif` without `else`, would be clearer

Reply:
Done.